statviz/src/views/Statistics/components/DemographicChart.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import { Heading } from "@chakra-ui/react";
+import BarChartCenterAxis from "../../../components/graphs/BarChartCenterAxis";
+import { range } from "lodash";
+import FilterCreatedOn from "./filter/FilterCreatedOn";
+import { HumanGender } from "../../../types/generated/graphql";
+
+export interface IDemographicFact {
```

Comment:
It is a bit weird that you have to define these types. Should the not also be generated by the codegen tool?

Something to check out in the future.

Reply:
Yes, the problem with the generated types is, that fields are nullable.
Since most fields there aren't null in any case i used this interface to not be forced to do null checks everywhere